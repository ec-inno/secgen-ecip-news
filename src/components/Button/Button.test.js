import React from 'react';
import renderer from 'react-test-renderer';
import Button from './Button';

describe('Button', () => {
  const props = {
    label: 'test',
  };

  it('renders correctly', () => {
    const tree = renderer.create(<Button {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
