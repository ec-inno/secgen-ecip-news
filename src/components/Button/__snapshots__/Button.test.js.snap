// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<button
  className="ecl-button ecl-button--primary"
  disabled={false}
  type="submit"
>
  <span
    className="ecl-button__container"
  >
    <span
      className="ecl-button__label"
      data-ecl-label={true}
    >
      test
    </span>
    
  </span>
</button>
`;
